tests/dapr_tests.mk

```diff
@@ -60,8 +60,6 @@ KUBECTL=kubectl
 
 DAPR_CONTAINER_LOG_PATH?=./dist/container_logs
 
-DAPR_TEST_SECONDARY_NAMESPACE=dapr-tests-2
```

Comment:
The secondary test namespace was not used anywhere, yet it contributed to challenges while running tests manually.